cmd/kube-apiserver/app/server.go

```diff
@@ -430,6 +430,7 @@ func BuildGenericConfig(s *options.ServerRunOptions) (*genericapiserver.Config,
 			versionedInformers.Core().V1().Services().Lister(),
 		)
 	}
+	genericConfig.SharedInformerFactory = versionedInformers
```

Comment:
@sttts Such `versionedInformers` is brought by #47857, which has been merged into all `v1.7.*` and `master` branches on June. So few people may get infected currently.

cc @deads2k Find your fingerprints. PTAL. Thanks.